server/src/utils/metas.ts

```diff
@@ -149,12 +149,15 @@ export async function replaceHtmlMetas(
   }
 }
 
-type NestedMetadata = string | number | boolean | { [key: string]: NestedMetadata }
+type NestedMetadata = string | number | boolean | string[] | { [key: string]: NestedMetadata }
 type Metadata = Record<string, NestedMetadata>
 
 const encodeWhateverType = (value: NestedMetadata): NestedMetadata => {
   switch (typeof value) {
     case 'object':
+      if (value instanceof Array) {
+        return value.map((item) => encode(item))
+      }
```

Comment:
Pas très fan de l'implem mais je vois pas comment on peut faire autrement

Reply:
```suggestion
      if (Array.isArray(value)) {
        return value.map((item) => encode(item))
      }
```

j'ai pas mieux à proposer